htdocs/core/class/commonobject.class.php

```diff
@@ -261,7 +261,8 @@ abstract class CommonObject
 	public $user;
 
 	/**
-	 * @var string 			The type of originating object ('commande', 'facture', ...). Note: on some object this field is called $origin_type
+	 * @var string|CommonObject			The type of the originating object ('commande', 'facture', ...). Note: on some object this field is called $origin_type
+	 * 									Or for MouvementStock, the originating object itself.
```

Comment:
In the past we may have an object inside $origin.
But now me should have only a string with type of object. Combined with $origin_id, we can recreate the object and save it into $origin_object.
So we should not have CommonObject anymore inside this.